lbry/lbry/extras/daemon/Daemon.py

```diff
@@ -3560,17 +3566,17 @@ def get_fee_address(self, kwargs: dict, claim_address: str) -> str:
             key, value = 'name', channel_name
         else:
             raise ValueError("Couldn't find channel because a channel_id or channel_name was not provided.")
-        for account in self.get_accounts_or_all(account_ids):
-            channels = await account.get_channels(**{f'claim_{key}': value}, limit=1)
-            if len(channels) == 1:
-                if for_signing and channels[0].private_key is None:
-                    raise Exception(f"Couldn't find private key for {key} '{value}'. ")
-                return channels[0]
-            elif len(channels) > 1:
-                raise ValueError(
-                    f"Multiple channels found with channel_{key} '{value}', "
-                    f"pass a channel_id to narrow it down."
-                )
+        all_accounts = self.get_accounts_or_all(account_ids)
+        channels = await self.ledger.db.get_channels(my_accounts=all_accounts, **{f'claim_{key}': value}, limit=1)
+        if len(channels) == 1:
+            if for_signing and channels[0].private_key is None:
+                raise Exception(f"Couldn't find private key for {key} '{value}'. ")
+            return channels[0]
+        elif len(channels) > 1:
+            raise ValueError(
+                f"Multiple channels found with channel_{key} '{value}', "
+                f"pass a channel_id to narrow it down."
+            )
```

Comment:
Did this method have to change? What's the functional difference between this and the previous version? Seems like previous version should still work.

Reply:
This addresses the ToDo that you told me to complete yesterday

Reply:
addressed*

Not sure if this will still work the new changes. Working on that.

Reply:
https://github.com/lbryio/lbry-sdk/pull/2220/commits/42064a09cc63c0f357f641b5f725b86cd5e0dffa#diff-dd8ed5f4b40f4021a557728289646ad7L3569